common/model/bundle.c

```diff
@@ -51,3 +51,55 @@ void calculate_bundle_hash(bundle_transactions_t *bundle, flex_trit_t *out) {
   flex_trits_from_trits(out, NUM_TRITS_HASH, bundle_hash_trits, NUM_TRITS_HASH,
                         NUM_TRITS_HASH);
 }
+
+void finalize_bundle(bundle_transactions_t *bundle) {
```

Comment:
could we call this `bundle_finalize`?

Reply:
sure, or `bundle_finalization`? b/c finalize is a verb.

Reply:
btw, `finalize_bundle` is the similar naming rule with `calculate_bundle_hash`.